rust/riscv64.patch

```diff
@@ -1,47 +1,81 @@
 diff --git PKGBUILD PKGBUILD
-index 8b39725..38d3c99 100644
+index 3e665b8a..a5da9820 100644
 --- PKGBUILD
 +++ PKGBUILD
 @@ -5,7 +5,7 @@
  # Contributor: userwithuid <userwithuid@gmail.com>
  
  pkgbase=rust
 -pkgname=(rust lib32-rust-libs rust-musl rust-wasm rust-src)
-+pkgname=(rust rust-wasm rust-src)
++pkgname=(rust rust-musl rust-wasm rust-src)
  epoch=1
- pkgver=1.58.1
+ pkgver=1.60.0
  pkgrel=1
-@@ -16,7 +16,7 @@ license=(MIT Apache)
+@@ -16,25 +16,31 @@ license=(MIT Apache)
  options=(!emptydirs !strip !lto)
- _llvm_ver=13.0.0
+ _llvm_ver=13.0.1
  depends=(gcc-libs llvm-libs curl libssh2 gcc)
 -makedepends=(rust "llvm=$_llvm_ver" libffi lib32-gcc-libs perl python cmake musl
+-             ninja wasi-libc lld)
 +makedepends=(rust "llvm=$_llvm_ver" libffi perl python cmake musl
-              ninja wasi-libc lld)
++             ninja wasi-libc lld git)
  checkdepends=(procps-ng gdb)
  source=(
-@@ -62,11 +62,9 @@ link-shared = true
+   "https://static.rust-lang.org/dist/rustc-$pkgver-src.tar.gz"{,.asc}
+   "https://github.com/llvm/llvm-project/releases/download/llvmorg-$_llvm_ver/compiler-rt-$_llvm_ver.src.tar.xz"{,.sig}
++  "git+https://github.com/rust-lang/libc.git#tag=0.2.116"
+   0001-cargo-Change-libexec-dir.patch
+   0001-bootstrap-Change-libexec-dir.patch
+   0002-compiler-Change-LLVM-targets.patch
+   0003-compiler-Use-wasm-ld-for-wasm-targets.patch
++  "use_local_libc_cc.patch"
```

Comment:
Looks like...this patch wasn't committed, and the committed "replace_patched_libc-cc.patch" wasn't used.